cat_server/helloworld/unikernel.ml

```diff
@@ -3,6 +3,9 @@ open V1_LWT
 
 module Main (C:CONSOLE) = struct
 
-  let start console = OS.Time.sleep 2.0 >>
+  let start console =
+    OS.Time.sleep 2.0
+
```

Comment:
Really? You're the first experience OCaml programmer to tell me that in >5 years :)
PR updated...